nvOC

```diff
@@ -210,16 +210,51 @@ Reboot()
    sudo su -c "shutdown -r now"
 }
 
+
+############################### Reboot host ###################################
+Reboot()
+{
+   # Reboots the host
+   echo -e "${B}${R}Warning: The host will reboot in 10 seconds, ctrl-c to cancel...${N}"
+   sleep 10
+   sudo su -c "shutdown -r now"
+}
+############################### Reboot host ###################################
+Restart()
+{
+   # Reboots the host
+        nvoc stop && timeout 2 && nvoc start
+}
 ############################### Start nvOC ####################################
 Start()
 {
-    if [ $LOCALorREMOTE == "LOCAL" ];then
-       gnome-terminal
-    else   
-       Reboot
-    fi   
+   if [ $LOCALorREMOTE == "LOCAL" ];then
+   pkill -f gnome-terminal
+   pkill -f 5watchdog
+   pkill -f 7telegram
+   pkill -f 6tempcontrol
+   pkill -e screen
+   pkill -f 8wtm
+      gnome-terminal -e "bash '/home/m1/3main'"
```

Comment:
This is wrong, here you should start a gnome-terminal on the default "mining" profile. Running 3main directly is not an option, it must always be running under 2unix running inside a re-up routine.

Reply:
I welcome your advice here Luke. How would the line be better written? gnome-terminal 'mining'? Not clued up on this.

Reply:
If you open the man page of gnome-terminal you'll see there is an option to mass either the profile name or id as an argument. You just need to open up a new window/tab on "mining" profile, that profile already exists in nvOC OS images by default.